Security agent relies on a dependency(check-disk-space) that doesn't support Node 14

[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

## Description
The Node.js agent has not dropped support for Node 14 just yet.  We got a [report from our community](https://github.com/newrelic/node-newrelic/issues/1725#issuecomment-1665461657) that the agent fails to install when `engine-strict=true` is provided to `npm install`.  They were able to trace this to the security agent and its dependency `check-disk-space`.  

## Steps to Reproduce
 1. On Node.js 14.x run `npm install @newrelic/security-agent --engine-strict=true`

## Expected Result
Security agent installs successfully

## Actual Result
Agent fails to install with following error

```sh
❯ npm i @newrelic/security-agent --engine-strict=true
npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!
npm ERR! code ENOTSUP
npm ERR! notsup Unsupported engine for check-disk-space@3.4.0: wanted: {"node":">=16"} (current: {"node":"14.21.3","npm":"6.14.18"})
npm ERR! notsup Not compatible with your version of node/npm: check-disk-space@3.4.0
npm ERR! notsup Not compatible with your version of node/npm: check-disk-space@3.4.0
npm ERR! notsup Required: {"node":">=16"}
npm ERR! notsup Actual:   {"npm":"6.14.18","node":"14.21.3"}

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/revans/.npm/_logs/2023-08-04T13_57_01_775Z-debug.log
```

--- a/THIRD_PARTY_NOTICES.md
+++ b/THIRD_PARTY_NOTICES.md
@@ -75,7 +75,7 @@ code, the source code can be found at [https://github.com/newrelic/csec-node-age
 
 ### @aws-sdk/client-lambda
 
-This product includes source derived from [@aws-sdk/client-lambda](https://github.com/aws/aws-sdk-js-v3) ([v3.358.0](https://github.com/aws/aws-sdk-js-v3/tree/v3.358.0)), distributed under the [Apache-2.0 License](https://github.com/aws/aws-sdk-js-v3/blob/v3.358.0/LICENSE):
+This product includes source derived from [@aws-sdk/client-lambda](https://github.com/aws/aws-sdk-js-v3) ([v3.379.1](https://github.com/aws/aws-sdk-js-v3/tree/v3.379.1)), distributed under the [Apache-2.0 License](https://github.com/aws/aws-sdk-js-v3/blob/v3.379.1/LICENSE):
 
 ```
                                 Apache License
@@ -311,7 +311,7 @@ THE SOFTWARE.
 
 ### check-disk-space
 
-This product includes source derived from [check-disk-space](https://github.com/Alex-D/check-disk-space) ([v3.4.0](https://github.com/Alex-D/check-disk-space/tree/v3.4.0)), distributed under the [MIT License](https://github.com/Alex-D/check-disk-space/blob/v3.4.0/LICENSE):
+This product includes source derived from [check-disk-space](https://github.com/Alex-D/check-disk-space) ([v3.3.1](https://github.com/Alex-D/check-disk-space/tree/v3.3.1)), distributed under the [MIT License](https://github.com/Alex-D/check-disk-space/blob/v3.3.1/LICENSE):
 
 ```
 MIT License
@@ -684,7 +684,7 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 
 ### semver
 
-This product includes source derived from [semver](https://github.com/npm/node-semver) ([v7.5.3](https://github.com/npm/node-semver/tree/v7.5.3)), distributed under the [ISC License](https://github.com/npm/node-semver/blob/v7.5.3/LICENSE):
+This product includes source derived from [semver](https://github.com/npm/node-semver) ([v7.5.4](https://github.com/npm/node-semver/tree/v7.5.4)), distributed under the [ISC License](https://github.com/npm/node-semver/blob/v7.5.4/LICENSE):
 
 ```
 The ISC License

--- a/package.json
+++ b/package.json
@@ -37,7 +37,7 @@
   "dependencies": {
     "@aws-sdk/client-lambda": "^3.363.0",
     "axios": "0.21.4",
-    "check-disk-space": "^3.4.0",
+    "check-disk-space": "3.3.1",
     "content-type": "^1.0.5",
     "fast-safe-stringify": "^2.1.1",
     "find-package-json": "^1.2.0",

--- a/third_party_manifest.json
+++ b/third_party_manifest.json
@@ -1,19 +1,19 @@
 {
-  "lastUpdated": "Sat Jun 24 2023 09:46:13 GMT+0530 (India Standard Time)",
+  "lastUpdated": "Fri Aug 04 2023 19:58:51 GMT+0530 (India Standard Time)",
   "projectName": "@newrelic/security-agent",
   "projectUrl": "https://github.com/newrelic/csec-node-agent.git",
   "includeOptDeps": false,
   "includeDev": true,
   "dependencies": {
-    "@aws-sdk/client-lambda@3.358.0": {
+    "@aws-sdk/client-lambda@3.379.1": {
       "name": "@aws-sdk/client-lambda",
-      "version": "3.358.0",
-      "range": "^3.348.0",
+      "version": "3.379.1",
+      "range": "^3.363.0",
       "licenses": "Apache-2.0",
       "repoUrl": "https://github.com/aws/aws-sdk-js-v3",
-      "versionedRepoUrl": "https://github.com/aws/aws-sdk-js-v3/tree/v3.358.0",
+      "versionedRepoUrl": "https://github.com/aws/aws-sdk-js-v3/tree/v3.379.1",
       "licenseFile": "node_modules/@aws-sdk/client-lambda/LICENSE",
-      "licenseUrl": "https://github.com/aws/aws-sdk-js-v3/blob/v3.358.0/LICENSE",
+      "licenseUrl": "https://github.com/aws/aws-sdk-js-v3/blob/v3.379.1/LICENSE",
       "licenseTextSource": "file",
       "publisher": "AWS SDK for JavaScript Team",
       "url": "https://aws.amazon.com/javascript/"
@@ -30,15 +30,15 @@
       "licenseTextSource": "file",
       "publisher": "Matt Zabriskie"
     },
-    "check-disk-space@3.4.0": {
+    "check-disk-space@3.3.1": {
       "name": "check-disk-space",
-      "version": "3.4.0",
-      "range": "^3.4.0",
+      "version": "3.3.1",
+      "range": "3.3.1",
       "licenses": "MIT",
       "repoUrl": "https://github.com/Alex-D/check-disk-space",
-      "versionedRepoUrl": "https://github.com/Alex-D/check-disk-space/tree/v3.4.0",
+      "versionedRepoUrl": "https://github.com/Alex-D/check-disk-space/tree/v3.3.1",
       "licenseFile": "node_modules/check-disk-space/LICENSE",
-      "licenseUrl": "https://github.com/Alex-D/check-disk-space/blob/v3.4.0/LICENSE",
+      "licenseUrl": "https://github.com/Alex-D/check-disk-space/blob/v3.3.1/LICENSE",
       "licenseTextSource": "file",
       "publisher": "Alex-D",
       "email": "contact@alex-d.fr",
@@ -211,15 +211,15 @@
       "email": "info@janogonzalez.com",
       "url": "http://janogonzalez.com"
     },
-    "semver@7.5.3": {
+    "semver@7.5.4": {
       "name": "semver",
-      "version": "7.5.3",
-      "range": "^7.5.3",
+      "version": "7.5.4",
+      "range": "^7.5.4",
       "licenses": "ISC",
       "repoUrl": "https://github.com/npm/node-semver",
-      "versionedRepoUrl": "https://github.com/npm/node-semver/tree/v7.5.3",
+      "versionedRepoUrl": "https://github.com/npm/node-semver/tree/v7.5.4",
       "licenseFile": "node_modules/semver/LICENSE",
-      "licenseUrl": "https://github.com/npm/node-semver/blob/v7.5.3/LICENSE",
+      "licenseUrl": "https://github.com/npm/node-semver/blob/v7.5.4/LICENSE",
       "licenseTextSource": "file",
       "publisher": "GitHub Inc."
     },

